src/status_im/common/terms/view.cljs

```diff
@@ -1,11 +1,15 @@
 (ns status-im.common.terms.view
   (:require-macros [legacy.status-im.utils.slurp :refer [slurp]])
-  (:require [quo.core :as quo]
+  (:require [clojure.string :as string]
+            [quo.core :as quo]
             [react-native.gesture :as gesture]
             [status-im.contexts.settings.common.header :as header]
             [utils.i18n :as i18n]))
 
-(def terms-of-use-text (slurp "resources/terms-of-use.mdwn"))
+(def terms-of-use-text
+  (-> (slurp "legal-docs/terms-of-use.md")
```

Comment:
Should this be `.mdwn` ?

Reply:
Update: It was decided that we could just update the current `.mdwn` file since it seems CI is not setup to pull down the contents of our submodule atm. So this change will be a piece of a different PR later.